### DISCUSSION TITLE
Who is your favorite Anime Character?

### DISCUSSION THREAD

**Discussion by ryosuke-shi** [NAIST-SE/AttractingContributors#1112] [2024 Attraction Contributions]
好きなアニメのキャラクターを教えてください！
Who is your favorite anime character? (It doesn't have to be a Japanese anime!)

link:
https://github.com/ryosuke-shi/AttractingContributors-AnimeCharacters

**Comment by ryosuke-shi** [comment]
Still need 7 people more. Please help me with this project🙇
ご協力よろしくお願いします<(_ _)>

**Comment by Canppchk** [comment]
I add Itachi Uchiha on your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen

**Reply by ryosuke-shi** [reply]
Thank you for your cooperation!
I have sent you a pull request.
